app/models/ability.rb

```diff
@@ -8,6 +8,10 @@ class Ability
 
   # Define any customized permissions here.
   def custom_permissions
+    if current_user.admin?
+      can [:create, :show, :add_user, :remove_user, :index, :edit, :update, :destroy], Role
```

Comment:
Ok, but the wiki page and gem haven't been updated since 2015, so not sure what that has to do with Sufia 7. But if that works now that I guess that's fine. There's this too: https://github.com/projecthydra/sufia/issues/2032